aries-site/src/data/structure.js

```diff
@@ -347,6 +377,10 @@ export const structure = [
     seoDescription:
       'When the user needs to select one or more options, use a checkbox.',
     sections: [],
+    previewComponent: {
+      component: () => <CheckBoxPreview />,
+      cardColor: 'background-front',
```

Comment:
It feels like most cards want the background to match the color of the content card. What if by default the previews used `background-front` (instead of `background-contrast`), then in the places we want to modify it and need background-contrast we can add it in with the cardColor. It mights save us some lines of code.